ProjectSettings/ProjectSettings.asset

```diff
@@ -650,14 +650,14 @@ PlayerSettings:
       HumanInterfaceDevice: False
       InputInjectionBrokered: False
       InternetClient: False
-      InternetClientServer: True
+      InternetClientServer: False
```

Comment:
Revert please. Sharing will stop working without these.

Reply:
I've no idea why these keep getting changed.  Will revert back to those in the dev working branch in a new PR.

Reply:
Resolved in #1648